models/glossary_test.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGlossarySortKeys(t *testing.T) {
+	glossary := Glossary{
+		"foo": "bar",
```

Comment:
[nit] Let's use "one" "two" "three"; sticks with existing pattern and is super easy to read the assertions below